modules/core/src/main/java/org/apache/ignite/internal/profiling/IgniteProfilingMBean.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.profiling;
+
+import org.apache.ignite.IgniteCheckedException;
+import org.apache.ignite.mxbean.MXBeanDescription;
+import org.apache.ignite.mxbean.MXBeanParameter;
+
+/**
+ * MBean provide access to profiling management.
+ */
+@MXBeanDescription("MBean provide access to profiling management.")
+public interface IgniteProfilingMBean {
+    /**
+     * Start profiling in the cluster with default settings.
+     *
+     * @see FileProfiling#DFLT_FILE_MAX_SIZE
+     * @see FileProfiling#DFLT_BUFFER_SIZE
+     * @see FileProfiling#DFLT_FLUSH_SIZE
+     */
+    @MXBeanDescription("Start profiling in the cluster.")
+    public void startProfiling() throws IgniteCheckedException;
```

Comment:
The word 'profiling' exists in each method of this bean. Can we remove it? `start`, `stop`, `enabled` seems less verbose.

Reply:
Fixed.